### PR TITLE
Add steps to release dgo client

### PR DESCRIPTION
**Pull request by mangalaman93**
This change is [Reviewable](https://reviewable.io/reviews/dgraph-io/dgo/121)

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+1. Update the `go.mod` file to the module name with the correct version
+
+2. Change all the import paths to import `v<new major version>`
+
+For example, current import path is `"github.com/dgraph-io/dgo/v200"`.
+When we release v201.07.0, we would replace the import paths to `"github.com/dgraph-io/dgo/v201"`
+
+3. Update [Supported Version](https://github.com/dgraph-io/dgo/#supported-versions)
+
+Now, follow the [steps to release Minor Version](#steps-to-release-a-minor-version) as above.
```
Comment (parasssh) [inline RELEASE.md:28]: Shall we mention to change import paths on Dgraph repo also?

Comment (mangalaman93) [reply RELEASE.md:30]: Done

File: RELEASE.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+When we release v201.07.0, we would replace the import paths to `"github.com/dgraph-io/dgo/v201"`
+
+3. Update [Supported Version](https://github.com/dgraph-io/dgo/#supported-versions)
+
+4. Commit all the changes and get them merged to master branch.
+
+Now, follow the [steps to release Minor Version](#steps-to-release-a-minor-version) as above.
+
+Note that, now you have to also change the import path in the applications that use `dgo` including `dgraph`.
+If you are doing the relese, it would be nice that you raise a PR on Dgraph repo as well updating the paths.
```
Comment (parasssh) [inline RELEASE.md:33]: spelling: relese -> release.
Sentence:  as well -> as well as

Comment (mangalaman93) [reply RELEASE.md:33]: Done, thanks, `as well` looks good I think

File: RELEASE.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+When we release v201.07.0, we would replace the import paths to `"github.com/dgraph-io/dgo/v201"`.
+
+3. Update [Supported Version](https://github.com/dgraph-io/dgo/#supported-versions).
+
+4. Commit all the changes and get them merged to master branch.
+
+Now, follow the [steps to release Minor Version](#steps-to-release-a-minor-version) as above.
+
+Note that, now you have to also change the import path in the applications that use `dgo` including `dgraph`.
+If you are doing the release, it would be nice that you raise a PR on Dgraph repo as well updating the paths.
```
Comment (parasssh) [inline RELEASE.md:33]: Reword as: 
Note that, now you may have to also change the import paths in the applications that use `dgo` including `dgraph` and raise appropriate PR for them.

**Pull request merged**